src/content/structured/community/roadmap.mdx

```diff
@@ -12,22 +12,67 @@ subTitle: "Our roadmap of recently released work and the work we have planned."
 contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/community/roadmap.mdx"
 ---
 
+## Introduction
+
+The roadmap shows what's new, what we're working on and our future plans.
+
+The purpose of the roadmap is to help teams understand when features are likely to be delivered. Some items might change due to priorities or organisation strategies.
+
+See our [GitHub](https://github.com/orgs/mi6/projects/2/views/1) for more details and our day-to-day activities.
+
 ## What's new
 
-- A public first for MI6, MI5 and GCHQ. We've just launched on [GitHub](https://github.com/mi6).
-- Intelligence Community Design System (ICDS) released as BETA.
-- Research, design and build of nearly 30 web components.
-- Official communications campaign from the United Kingdom Intelligence Community (UKIC).
+We recently added:
+
+- Toasts.
+- Cards V2, with images and expandable area.
+- Asynchronous filtering for search bar.
+- Compact stepper.
+- Chips.
+- Popover menu.
 
 ## Working on now
 
-- Establishing our presence as we learn to work in the public.
-- Analytics to help with metrics gathering for launch.
-- Releasing our Figma component library, includes publishing to Figma Community.
+We're working to:
+
+- Release our Figma component library, including publishing to Figma Community.
+- Make sustainability improvements.
+
+As well as produce the following components:
+
+- Data table.
+- Date input.
+- Dialog.
+- Pagination.
+- Accordion.
+- Drawer.
+- Filters.
+- Clickable chips.
+
+## Coming up next
+
+We're getting ready to:
+
+- Investigate WCAG 2.2 changes.
+- Implement dark mode styling.
+
+As well as produce the following components:
+
+- Data visualisation.
+- Tree view.
+- Tecnhical utility components, such as no SSR and no JS.
 
 ## Future plans
 
-- Further research for complex components like data tables, date pickers and accessible charts and graphs.
-- Promote this work wider across central UK Government.
-- Improve codebase of this Design System website.
-- Refresh 'patterns' section with latest user research.
+We plan to:
+
+- Investigate customisation of our components.
+- Create app templates.
+
+As well as produce the following components:
+
+- WYSIWYG editor.
+- Video player.
+- Timeline.
+- Powerbar.
+- Mapping.
```

Comment:
```suggestion
- Video player,
- Timeline,
- Powerbar,
- Mapping.
```

Reply:
Not sure it makes sense to end on commas as they're technically all different items.